### PR TITLE
Changes added to Comb_Sort

### LINKED ISSUE #2135: Comb Sort
- [x] C
- [x] C++
- [ ] C# - @sakshijainn
- [ ] CoffeeScript
- [ ] Dart
- [x] Go
- [x] Java
- [ ] JS - @atrip10
- [x] Kotlin - @sandhyabhan
- [x] Php
- [x] Python
- [x] Readme
- [ ] Ruby
- [ ] Rust
- [ ] TypeScript

### PR DESCRIPTION
**Pull request by atrip10**
Fixes #2135 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- 
- 

#### Languages Used:
- 
- 

#### Files Added:
- 
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by raghav-dalmia** [comment]
Add issue number in PR description and squash your commits.

**Comment by raghav-dalmia** [comment]
Your PR has one extra file. Remove it.

**Comment by atrip10** [comment]
> Your PR has one extra file. Remove it.

How to remove it?

File: Comb_Sort/comb_sort.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+function combSort(arr)
+{
+    // the shrink factor is taken to be 1.3
+    var gap = Math.floor(arr.length / 1.3);
+    
+    /* loop to reduce gap and implement sorting */ 
+    while(gap > 0){
+        for(var i=0; i+gap < arr.length; i++){
+            // swap the values using a temporary variable
+                if(arr[i] > arr[i+gap]){
```
Comment (SakshiShreya) [inline Comb_Sort/comb_sort.js]: Correct the indentation here.

File: Comb_Sort/comb_sort.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+Function to implement Comb sort using JavaScript
+Example : If input array is [9,7,8,3,2], then after implementing this code the output array will be [2,3,7,8,9].  
+*/
+
+function combSort(arr)
+{
+    // the shrink factor is taken to be 1.3
+    var gap = Math.floor(arr.length / 1.3);
+    
+    /* loop to reduce gap and implement sorting */
```
Comment (SakshiShreya) [inline Comb_Sort/comb_sort.js]: This can be a single line comment.

File: Comb_Sort/comb_sort.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+Example : If input array is [9,7,8,3,2], then after implementing this code the output array will be [2,3,7,8,9].  
+*/
+
+function combSort(arr)
+{
+    // the shrink factor is taken to be 1.3
+    var gap = Math.floor(arr.length / 1.3);
+    
+    /* loop to reduce gap and implement sorting */ 
+    while(gap > 0){
```
Comment (SakshiShreya) [inline Comb_Sort/comb_sort.js]: Either follow
```
keyword()
{
```
this or 
```
keyword() {
```

You have used the combination of both.

If you want to use the second option, then add a space between `)` and `{`

File: Comb_Sort/comb_sort.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        for(var i=0; i+gap < arr.length; i++){
+            // swap the values using a temporary variable
+                if(arr[i] > arr[i+gap]){
+                    var next = arr[i+gap];
+                    arr[i+gap] = arr[i];
+                    arr[i] = next;
+                 }
+        }
+        gap=Math.floor(gap / 1.3);
```
Comment (SakshiShreya) [inline Comb_Sort/comb_sort.js]: Add space around operators.

File: Comb_Sort/comb_sort.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    }
+    return arr;
+};
+
+/* psuedo data for array */
+var arr = [3,9,8,2,5,1];
```
Comment (SakshiShreya) [inline Comb_Sort/comb_sort.js]: Add space after comma

File: Comb_Sort/comb_sort.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+                    arr[i+gap] = arr[i];
+                    arr[i] = next;
+                 }
+        }
+        gap=Math.floor(gap / 1.3);
+    }
+    return arr;
+};
+
+/* psuedo data for array */
```
Comment (SakshiShreya) [inline Comb_Sort/comb_sort.js]: This can also be a single line comment

**Comment by somya-kapoor** [comment]
@Are you working on it?

**Comment by somya-kapoor** [comment]
Due to no activity for more than 7 days, I am closing this issue.

**Pull request closed without merging**